app/src/main/java/com/dimelo/sampleapp/chats/TabTelecom.java

```diff
@@ -84,12 +88,44 @@ private void customize(){
+        customisation.agentTimeColor = Color.GREEN;
         customisation.systemMessageTextColor = Color.BLACK;
-        customisation.dateTextColor = Color.WHITE;
+        customisation.dateTextColor = Color.RED;
+        customisation.dateFont = Typeface.DEFAULT_BOLD;
         customisation.hourTimeTextColor = Color.WHITE;
         customisation.createNewThreadBackgroundColor = Color.RED;
+        int paddingReplies = (int)TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_padding), getResources().getDisplayMetrics());
+        customisation.quickRepliesItemPadding = new RcFragment.Customization.Padding(paddingReplies, paddingReplies, paddingReplies, paddingReplies);
+        SimpleDateFormat dateFormatter = new SimpleDateFormat("yyyy-MM-dd");
+        customisation.dateFormatter = dateFormatter;
+        customisation.messageFont = Typeface.DEFAULT_BOLD;
+        customisation.agentTemplateBorderColor = Color.RED;
+        customisation.agentTemplateWithImageBodyBackgroundColor = Color.LTGRAY;
+        int quickRepliesHeight = (int)TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_height), getResources().getDisplayMetrics());
+        customisation.quickRepliesBorderWidth = quickRepliesHeight;
+        customisation.quickRepliesTextColor = Color.WHITE;
+        customisation.quickRepliesTappedTextColor = Color.RED;
+        customisation.quickRepliesBorderColor = Color.RED;
+        customisation.badgeFont = Typeface.DEFAULT_BOLD;
+        customisation.badgeTextColor = Color.RED;
+        customisation.backToAllChatsFont = Typeface.DEFAULT_BOLD;
+        customisation.threadsListSeparatorColor = Color.BLACK;
+        customisation.quickRepliesHorizontalSpacing = (int)TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, getResources().getDimension(R.dimen.rc_quick_replies_horizontal_spacing), getResources().getDisplayMetrics());
+        customisation.hourTimeFont = Typeface.DEFAULT_BOLD;
+        customisation.agentTimeFont = Typeface.DEFAULT_BOLD;
+        customisation.hourTimeTextColor = Color.GREEN;
+        customisation.agentStructuredMessageItemFont = Typeface.DEFAULT_BOLD;
+        customisation.agentStructuredMessageUrlFont = Typeface.DEFAULT_BOLD;
+        customisation.agentStructuredMessageSubTitleFont = Typeface.DEFAULT_BOLD;
+        customisation.agentStructuredMessageTitleFont = Typeface.DEFAULT_BOLD;
+        customisation.agentStructuredMessageTitleColor = Color.BLUE;
+        customisation.agentStructuredMessageSubtitleColor = Color.RED;
+        customisation.agentStructuredMessageUrlColor = Color.RED;
+        customisation.agentStructuredMessageItemColor = Color.RED;
+        customisation.agentStructuredMessageItemTappedColor = Color.BLUE;
+        customisation.setBackToAllChatsImage(R.drawable.back_all_chat);
+        customisation.setLockedThreadImage(R.drawable.back_all_chat);
```

Comment:
same remark

Reply:
and you can remove the `back_all_chat` image

Reply:
ok